scripts.js

```diff
@@ -138,17 +130,13 @@ function getRandomIndex(hexCodeArray) {
 }
 
 function changeHex(currentPalette) {
-  hex1.innerText = currentPalette.colorPalette[0].ranHexCode
-  hex2.innerText = currentPalette.colorPalette[1].ranHexCode
-  hex3.innerText = currentPalette.colorPalette[2].ranHexCode
-  hex4.innerText = currentPalette.colorPalette[3].ranHexCode
-  hex5.innerText = currentPalette.colorPalette[4].ranHexCode
+  for (var i=0; i<=4; i++) {
+    allHex[i].innerText = currentPalette.colorPalette[i].ranHexCode
+  }
 }
 
 function background(currentPalette) {
-  color1.style.backgroundColor = currentPalette.colorPalette[0].ranHexCode
-  color2.style.backgroundColor = currentPalette.colorPalette[1].ranHexCode
-  color3.style.backgroundColor = currentPalette.colorPalette[2].ranHexCode
-  color4.style.backgroundColor = currentPalette.colorPalette[3].ranHexCode
-  color5.style.backgroundColor = currentPalette.colorPalette[4].ranHexCode
+  for (var i=0; i<=4; i++) {
+    allColor[i].style.backgroundColor = currentPalette.colorPalette[i].ranHexCode
+  }
 }
```

Comment:
The for loop used is a more efficient use of code. This may be able to be used for other functions.